amr-wind/wind_energy/ABLFieldInit.H

```diff
@@ -47,6 +47,20 @@ public:
     void init_tke(const amrex::Geometry& geom, amrex::MultiFab& tke) const;
 
 private:
+    // ABL-with-bubble
```

Comment:
Is the bubble thing something you want to keep? Or was this just a debug thing? If you want to keep it, it might be better to move this to the `physics` folder/class. Would you be ok with that?

Reply:
It is a debug thing where Bruce was using temperature in lieu of a scalar. I would like to use the scalar instead so I'll make that change at some point.